clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -30,29 +33,48 @@ const ALL_COLUMNS: ColumnMetadata[] = [
   { name: "Resource ID", attribute: "fidesctl_meta.resource_id" },
 ];
 
-const ScanResultsForm = () => {
+interface Props {
+  manualSystemSetupChosen: boolean;
+}
+
+const ScanResultsForm = ({ manualSystemSetupChosen }: Props) => {
   const systems = useAppSelector(selectSystemsForReview);
   const dispatch = useAppDispatch();
+  const router = useRouter();
   const {
     isOpen: isWarningOpen,
     onOpen: onWarningOpen,
     onClose: onWarningClose,
   } = useDisclosure();
+  const [createSystem] = useCreateSystemMutation();
   const [selectedSystems, setSelectedSystems] = useState<System[]>(systems);
+  const features = useFeatures();
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const createSystems = async () => {
+    await selectedSystems.forEach((system: System) => createSystem(system));
+  };
```

Comment:
I don't think we should do a `forEach` here. We should use the system `/upsert` endpoint instead, which allows creating multiple systems at once. You can see [how it's used in plus](https://github.com/ethyca/fidesctl-plus/blob/main/clients/datamap-ui/src/features/scan/scan.slice.ts#L38-L49) for reference

Reply:
Thank you! Keeping Upserts in my back pocket again 💃